inventory/staging/hosts

```diff
@@ -1,8 +1,12 @@
 [tag_app_mesosmasterstaging]
 [tag_app_mesosslavestaging]
+[tag_app_glusterstaging]
 
 [mesos-master:children]
 tag_app_mesosmasterstaging
 
 [mesos-slave:children]
 tag_app_mesosslavestaging
+
+[storage:children]
+tag_app_glusterstaging
```

Comment:
This should be changed to just tag_app_gluster since I don't think we need two separate storage clusters (See the Discourse thread).